modules/logging-delivery-tuning.adoc

```diff
@@ -0,0 +1,104 @@
+
+[IMPORTANT]
+====
+To use this feature, your {logging} deployment must be configured to use the Vector collector. The `tuning` spec in the `ClusterLogForwarder` CR is not supported when using the Fluentd collector.
+====
+
+The following example shows the `ClusterLogForwarder` CR options that you can modify to tune log forwarder outputs:
+
+.Example `ClusterLogForwarder` CR tuning options
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+# ...
+spec:
+  tuning:
+      delivery: AtLeastOnce # <1>
+      compression: none # <2>
+      maxWrite: <integer> # <3>
+      minRetryDuration: 1s # <4>
+      maxRetryDuration: 1s # <5>
+# ...
+----
+<1> Specify the delivery mode for log forwarding. The default delivery mode value is `AtLeastOnce`.
+** `AtLeastOnce` delivery means that if the log forwarder crashes or is restarted, any logs that were read before the crash but not sent to their destination are re-sent. It is possible that some logs are duplicated after a crash.
+** `AtMostOnce` delivery means that the log forwarder makes no effort to recover logs lost during a crash. This mode gives better throughput, but may result in greater log loss.
+<2> Specifying a `compression` configuration causes data to be compressed before it is sent over the network. Note that not all output types support compression, and if the specified compression type is not supported by the output, this results in an error. The possible values for this configuration are `none` for no compression, `gzip`, `snappy`, `zlib`, or `zstd`. `lz4` compression is also available if you are using a Kafka output. See the table "Supported compression types for tuning outputs" for more information.
+<3> Specifies a limit for the maximum payload of a single send operation to the output.
+<4> Specifies a minimum duration to wait between attempts before retrying delivery after a failure. This value is a string, and can be specified as milliseconds (`ms`), seconds (`s`), or minutes (`m`).
+<5> Specifies a maximum duration to wait between attempts before retrying delivery after a failure. This value is a string, and can be specified as milliseconds (`ms`), seconds (`s`), or minutes (`m`).
+
+.Supported compression types for tuning outputs
+[options="header"]
+|===
+|Compression algorithm |Splunk |Amazon Cloudwatch |Elasticsearch 8 |LokiStack |Apache Kafka |HTTP |Syslog |Google Cloud |Microsoft Azure Monitoring
+
+|`gzip`
+|X
+|X
+|X
+|X
+|
+|X
+|
+|
+|
+
+|`snappy`
+|
+|X
+|
+|X
+|X
+|X (Vector only)
+|
+|
+|
+
+|`zlib`
+|
+|X
+|X
+|
+|
+|X (Fluentd only)
```

Comment:
Please remove  the workd (Fluentd only), (Vector only).   Fluentd, vector can act as a httpserver.  I am using vector or fluentd in my test. We needn't specify the httpserver types here.

Reply:
I figured but kept it anyway since it was in the spreadsheet :) will remove